src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -56,6 +59,39 @@ namespace Utils
         {
         }
 
+        /**
+         * @brief Class destructor. Frees column family handles.
+         *
+         */
+        ~RocksDBWrapper()
+        {
+            std::for_each(m_handles.begin(),
+                          m_handles.end(),
+                          [this](rocksdb::ColumnFamilyHandle* handle) { m_db->DestroyColumnFamilyHandle(handle); });
+        }
+
+        /**
+         * @brief Creates a new column family in the database.
```

Comment:
Improve (documentation): It also adds the handle to the m_handles member

Reply:
Done!